package.json

```diff
@@ -6,7 +6,7 @@
   "type": "module",
   "scripts": {
     "sushi": "node node_modules/.bin/sushi fsh-tank",
-    "test-cql": "./node_modules/.bin/mocha --reporter spec --recursive --timeout 100000",
+    "test-cql": "C:/Users/ptran/Desktop/ccsm-cds-with-tests/node_modules/.bin/mocha --reporter spec --recursive --timeout 100000",
```

Comment:
I think there's a way to do this on windows that doesn't involve specifying the full path.

Reply:
@dwwinters The `test-cql-windows` script that you added in the next commit works on my Mac. Is there any reason we can't just make this the default `test-cql`, so that we don't have two OS-specific scripts?

If not, then I think we should at least mention `test-cql-windows` in the "Run CQL Tests" section of the README.

Reply:
```"test-cql": "mocha --reporter spec --recursive --timeout 100000", ``` also works on my windows, if we are deciding to not have two OS-specific scripts

Reply:
If it works on both, then I don't see why not just have one script.  Can't remember my reasoning there...

Reply:
Great! @ptran265 do you want to make the update to remove the unnecessary `test-cql-windows` script?